runPerftool.psm1

```diff
@@ -32,6 +32,19 @@ Param ([Int]$AdditionalTimeout, [string] $Line)
         }
        catch {}
     }
+    elseif ($Line -match "secnetperf") {
```

Comment:
nit...  if not manipulating a string, use single quotes ( this applies to multiple places in this review, non blocking, but good habit to get into )